java/client/src/main/java/glide/api/models/commands/scan/ClusterScanCursor.java

```diff
@@ -0,0 +1,92 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.scan;
+
+import glide.api.commands.GenericClusterCommands;
+
+/**
+ * A cursor is used to iterate through data returned by cluster SCAN requests.
+ *
+ * @see GenericClusterCommands#scan}.
```

Comment:
```suggestion
 * @see {@link GenericClusterCommands#scan}.
```

Reply:
This seems to trigger an IntelliJ warning:
Class/method reference, quoted text, or HTML link are expected after @see tag

Reply:
Is your code in a valid javadoc?  I admit, I didn't know you could only needed one `}`.
```
 * @see GenericClusterCommands#scan}.
```